index.html

```diff
@@ -71,9 +71,7 @@ <h2 id="technology">Technology-specific</h2>
     <h2 id="beginner">Beginner-friendly coding meetups</h2>
     <ul>
       <li><a href="https://codebar.io/">Codebar</a> - workshops and meetups with coaches to help you with your own projects, or guide you through their coding tutorials (with a goal to enable underrepresented groups to learn programming).</li>
-      <li><a href="https://www.facebook.com/groups/KeepCalmAndCarryOnCoding/" title="Keep Calm and Carry on Coding facebook group">Keep Calm and Carry on Coding</a> - a fortnightly meetup aimed at anyone have who has recently done a coding course and wants to continue learning.</li>
-      <li><a href="http://codeandcake.club">Code &amp;&amp; Cake</a> - regular meetups where you can work on your own coding side projects whilst eating cake (hurrah!); open to all experience levels.</li>
-      <li><a href="http://www.meetup.com/founderscoders/events/225965453/" title="Founders and Coders Coding for Everyone">Coding for Everyone (Founders &amp; Coders)</a> - find someone of a similar level to you to talk through and work on code with.</li>
+      <li><a href="https://www.meetup.com/founderscoders/" title="Founders and Coders Coding for Women">Coding for Women (Founders &amp; Coders)</a> - find someone of a similar level to you to talk through and work on code with.</li>
```

Comment:
Suggest we take out the title attribute here - it doesn't add additional information for the user; sorry that the example you had to go on also had an unnecessary title attribute!

```suggestion
      <li><a href="https://www.meetup.com/founderscoders/">Coding for Women (Founders &amp; Coders)</a> - find someone of a similar level to you to talk through and work on code with.</li>
```

Reply:
Based on the discussion in issue #8, as opposed to this change, we can just remove this line too 👍